spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/AboutController.java

```diff
@@ -149,23 +149,27 @@ public AboutResource getAboutResource() {
 		final RuntimeEnvironment runtimeEnvironment = new RuntimeEnvironment();
 
 		if (this.streamDeployer != null) {
-			final RuntimeEnvironmentInfo deployerEnvironmentInfo = this.streamDeployer.environmentInfo();
-			final RuntimeEnvironmentDetails deployerInfo = new RuntimeEnvironmentDetails();
-
-			deployerInfo.setDeployerImplementationVersion(deployerEnvironmentInfo.getImplementationVersion());
-			deployerInfo.setDeployerName(deployerEnvironmentInfo.getImplementationName());
-			deployerInfo.setDeployerSpiVersion(deployerEnvironmentInfo.getSpiVersion());
-			deployerInfo.setJavaVersion(deployerEnvironmentInfo.getJavaVersion());
-			deployerInfo.setPlatformApiVersion(deployerEnvironmentInfo.getPlatformApiVersion());
-			deployerInfo.setPlatformClientVersion(deployerEnvironmentInfo.getPlatformClientVersion());
-			deployerInfo.setPlatformHostVersion(deployerEnvironmentInfo.getPlatformHostVersion());
-			deployerInfo.setPlatformSpecificInfo(deployerEnvironmentInfo.getPlatformSpecificInfo());
-			deployerInfo.setPlatformHostVersion(deployerEnvironmentInfo.getPlatformHostVersion());
-			deployerInfo.setPlatformType(deployerEnvironmentInfo.getPlatformType());
-			deployerInfo.setSpringBootVersion(deployerEnvironmentInfo.getSpringBootVersion());
-			deployerInfo.setSpringVersion(deployerEnvironmentInfo.getSpringVersion());
-
-			runtimeEnvironment.setAppDeployer(deployerInfo);
+			try {
+				final RuntimeEnvironmentInfo deployerEnvironmentInfo = this.streamDeployer.environmentInfo();
+				final RuntimeEnvironmentDetails deployerInfo = new RuntimeEnvironmentDetails();
+
+				deployerInfo.setDeployerImplementationVersion(deployerEnvironmentInfo.getImplementationVersion());
+				deployerInfo.setDeployerName(deployerEnvironmentInfo.getImplementationName());
+				deployerInfo.setDeployerSpiVersion(deployerEnvironmentInfo.getSpiVersion());
+				deployerInfo.setJavaVersion(deployerEnvironmentInfo.getJavaVersion());
+				deployerInfo.setPlatformApiVersion(deployerEnvironmentInfo.getPlatformApiVersion());
+				deployerInfo.setPlatformClientVersion(deployerEnvironmentInfo.getPlatformClientVersion());
+				deployerInfo.setPlatformHostVersion(deployerEnvironmentInfo.getPlatformHostVersion());
+				deployerInfo.setPlatformSpecificInfo(deployerEnvironmentInfo.getPlatformSpecificInfo());
+				deployerInfo.setPlatformHostVersion(deployerEnvironmentInfo.getPlatformHostVersion());
+				deployerInfo.setPlatformType(deployerEnvironmentInfo.getPlatformType());
+				deployerInfo.setSpringBootVersion(deployerEnvironmentInfo.getSpringBootVersion());
+				deployerInfo.setSpringVersion(deployerEnvironmentInfo.getSpringVersion());
+
+				runtimeEnvironment.setAppDeployer(deployerInfo);
+			} catch (UnsupportedOperationException uoe) {
+				logger.debug("expected in skipper mode due to #392");
```

Comment:
This message can be generic as there could be other modes in the future. Maybe we don't need to specify Skipper's 392 here.

Reply:
+1